masonry/src/widget/checkbox.rs

```diff
@@ -30,15 +30,15 @@ impl Checkbox {
     pub fn new(checked: bool, text: impl Into<ArcStr>) -> Checkbox {
         Checkbox {
             checked,
-            label: WidgetPod::new(Label::new(text)),
+            label: WidgetPod::new(Label::new(text).with_skip_pointer(true)),
```

Comment:
When would a label not skip pointer handling?

I guess maybe if we get link support within labels?

Reply:
Links and selecting text (though I think maybe we have the Prose widget for that now?).

I guess we could go ahead and say they're *always* skipped and only Prose gets to have links. I haven't really thought this through and I don't think it's a priority.